exercises/decimal/src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub struct Decimal {
 }
 
 impl Decimal {
-    pub fn try_from(_: &str) -> Option<Decimal> {
-        unimplemented!()
+    pub fn try_from(input: &str) -> Option<Decimal> {
+        unimplemented!("Create a new decimal with a value of {:?}", input)
```

Comment:
I think I prefer you use `{}` here, since it will say `Create a new decimal with a value of 1.1` rather than `Create a new decimal with a value of "1.1"` and I'd rather see it without the quotes.